src/test/java/teammates/test/cases/datatransfer/CourseAttributesTest.java

```diff
@@ -20,13 +20,15 @@
     private String validId = "validId";
     private ZoneId validTimeZone = ZoneId.of("UTC");
     private Instant validCreatedAt = Instant.ofEpochMilli(98765);
+    private Instant validDeletedAt = Instant.ofEpochMilli(99999);
```

Comment:
I assume `validDeletedAt` must be after `validCreatedAt` for these instants to make semantic sense, right? If so, define `validDeletedAt` relative to `validCreatedAt` in case someone tampers with the `98765`. Something like` validDeletedAt = validCreatedAt.plusSeconds(1000)`.